src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/SendNotificationHandler.java

```diff
@@ -72,7 +72,8 @@ public boolean canHandle(
                    Event.REMOVE_APPEAL_FROM_ONLINE,
                    Event.CHANGE_HEARING_CENTRE,
                    Event.APPLY_FOR_FTPA_APPELLANT,
-                   Event.APPLY_FOR_FTPA_RESPONDENT
+                   Event.APPLY_FOR_FTPA_RESPONDENT,
+                   Event.RESTORE_STATE_FROM_ADJOURN
```

Comment:
Are we testing for the new event in send notifications handler?

Reply:
done. Thanks!